scripts/semver-checks.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -x
+
+cargo semver-checks -p scylla -p scylla-cql $@
```

Comment:
proc-macros crates are not checked by cargo-semver-checks. If you run just `cargo semver-checks` it will check scylla, scylla-cql, scylla-proxy, but not scylla-macros.